colossalai/checkpoint_io/utils.py

```diff
@@ -88,6 +90,17 @@ def is_safetensor_checkpoint(checkpoint_file_path: str) -> bool:
 # ======================================
 # Helper functions for saving shard file
 # ======================================
+def unwrap_optimizer(optimizer: OptimizerWrapper):
+    '''
+    Unwrap a wrapped optimizer.
+    This method should be used before saving/loading it to/from sharded checkpoints.
+    '''
```

Comment:
Please add a TODO comment here. `ColossalaiOptimizer` will be replaced with `OptimizerWrapper` in the future.

Reply:
Updated